### ISSUE TITLE
Publication not found

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1301]
A user point to this: 
"https://oerworldmap.org/resource/?q=Wikisource%3A+Tourenb%C3%BCcher+f%C3%BCr+Radfahrer brint keine Treffer.
https://oerworldmap.org/resource/?q=Tourenb%C3%BCcher+f%C3%BCr+Radfahrer bringt vier Treffer."
I do understand the 4 results but why is there noresult in the first place?

**Comment by literarymachine** [comment]
> I do understand the 4 results but why is there noresult in the first place?

Our queries support the [Elasticsearch URI search syntax](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-uri-request.html). In this, the word before a colon is interpreted as a field name; since there is no field named "Wikisource" in our data, nothing is found. If you escape the colon, search results are as expected: https://oerworldmap.org/resource/?q=Wikisource%5C%3A+Tourenb%C3%BCcher+f%C3%BCr+Radfahrer

**Comment by karindr** [comment]
OK, I understand this. Bu I think the users expect our search to behave like the google search and using a : there makes no difference.
For what purposes do we need to have this URI search? Can we get the results in another way and change the behaviour of our search to ignoring the :?

**Comment by literarymachine** [comment]
> For what purposes do we need to have this URI search?

We have it to be able to search only certain fields. And also e.g. for various occasions when checking if a field exists as in https://oerworldmap.org/resource/?q=_exists_:about.license&filter.about.@type=Service

**Comment by acka47** [comment]
I agree with Karin that we should either automatically escape or remove colons from terms entered into the search field.

**Comment by literarymachine** [comment]
> I agree with Karin that we should either automatically escape or remove colons from terms entered into the search field.

BTW, not escaping is the exact behavior in lobid.org: http://lobid.org/resources/search?q=Foto:Box

**Comment by acka47** [comment]
@literarymachine discussed this on slack and came to the conclusion that a new search parameter that automatically escapes search terms so that they don't make problems with ES is the best way to go. The disadvantage is, though, that one can not anymore enter complex queries in the search field.

**Comment by literarymachine** [comment]
>  The disadvantage is, though, that one can not anymore enter complex queries in the search field.

Instead of adding a new query parameter, we could also define a prefix for "power user mode", e.g. `::`:

`::field:foo AND _exists_:bar`

@acka47 what do you think?
